eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/CommandCodecs.scala

```diff
@@ -88,19 +103,36 @@ object CommandCodecs {
   private val cmdFailWithoutHoldTimeCodec: Codec[CMD_FAIL_HTLC] =
     (("id" | int64) ::
       ("reason" | failureReasonCodec) ::
-      ("htlcReceivedAt_opt" | provide(Option.empty[TimestampMilli])) ::
+      ("attribution_opt" | provide(Option.empty[FailureAttributionData])) ::
       // No need to delay commands after a restart, we've been offline which already created a random delay.
       ("delay_opt" | provide(Option.empty[FiniteDuration])) ::
       ("commit" | provide(false)) ::
       ("replyTo_opt" | provide(Option.empty[ActorRef]))).as[CMD_FAIL_HTLC]
 
-  private val cmdFailCodec: Codec[CMD_FAIL_HTLC] =
+  private val cmdFailWithHoldTimeCodec: Codec[CMD_FAIL_HTLC] =
```

Comment:
`WithHoldTime` is a weird name given that it doesn't have any hold time. Why not `WithPartialAttribution` like you did for the fulfill?